File: src/vulnerable.java
// Vuln.java
import java.sql.*;
import java.io.*;

public class Vuln {
    public static void main(String[] args) throws Exception {
        String user = args[0];

        Statement stmt = DriverManager
            .getConnection("jdbc:mysql://localhost/test", "root", "root")
            .createStatement();

        stmt.execute("SELECT * FROM users WHERE name = '" + user + "'"); // SQL Injection

        Runtime.getRuntime().exec(user); // Command Injection
    }
}

File: src/vulnerable.c
// vuln.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(int argc, char *argv[]) {
    char buffer[10];
    strcpy(buffer, argv[1]); // Buffer Overflow

    system(argv[1]); // Command Injection
    return 0;
}
